distribution/docs/src/main/resources/content/_configuring/map-layers-standard.adoc

```diff
@@ -0,0 +1,30 @@
+:title: Configuring Map Layers for ${ddf-ui}
+:type: subConfiguration
+:status: published
+:parent: Configuring ${ddf-ui}
+:order: 00
+:summary: Configuring map layers for ${ddf-ui}.
+
+== {title}
+
+Customize the look of the map displayed to users in ${ddf-ui} by adding or removing map layers through the Map Layers UI.
+
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-ui}* application.
+. Select the *Configuration* tab.
+. Select the *Standard Search UI* configuration.
+
+.Adding a Map Layer
+. Select the *add* button to add a new layer
```

Comment:
Missing a period at the end of this sentence

Reply:
fixed.